css/selectors4/focus-within-shadow-001.html

```diff
@@ -6,7 +6,7 @@
 <link rel="author" title="Florian Rivoal" href="mailto:florian@rivoal.net">
 <link rel="help" href="https://drafts.csswg.org/selectors-4/#focus-within-pseudo">
 <link rel="match" href="focus-within-shadow-001-ref.html">
-<meta name="flags" content="interact dom">
```

Comment:
Similarly to the comment on focus-within-001:

As we switch to js required, it would be good to add `class="reftest-wait"` on the root, and remove it once js has run, so that we make sure that the ref comparison is done at the right time, and that if js fails to run for some reason, we detect it as a test that failed to run rather than a failure to implement focus-within correctly.

I note that this test already has that class on the `#focusme` element, but that's not where it is supposed to be, so that's a mistake that survived the previous review. Would be good to fix while we're at it.